LoungewareNoNoGenerator/Program.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
+using System.Linq;
+
+//you can access tha command line args in VS by right clicking the project > properties > debug > command line args
+//alternatively run with dotnet run <args> after cd'ed into LoungewareNoNoGenerator/LoungewareNoNoGenerator
+
+//output will be in bin/Debug/net5.0
+
+//args format:
+//<func_name> <use_instead> ...
+//pass undefined for <use_instead> if there's not an alternative
+//e.g. dotnet run game_end undefined audio_play_sound sfx_play
+
+IEnumerable<string> insteads = args.Where((val, index) => index % 2 == 1);
+IEnumerable<(string Func, string Instead)> funcs = args
+    .Where((val, index) => index % 2 == 0)
+    .Zip(insteads);
```

Comment:
perhaps a text file could be read instead of passing command-line arguments, so that other developers can add new commands to ignore relatively simply